scripts/deploy_browser_prod_manual.sh

```diff
@@ -30,6 +30,7 @@ cp "build/${NAME}-${CURRENT_VERSION}.js" "cdn/static/javascript/${NAME}-${CURREN
 cp "build/${NAME}-${CURRENT_VERSION}.min.js" "cdn/static/javascript/${NAME}-${CURRENT_VERSION_MAJOR}.min.js"
 
 cd cdn
+git checkout -b ${NAME}-${CURRENT_VERSION}
```

Comment:
It's best to always quote variables.
```suggestion
git checkout -b "${NAME}-${CURRENT_VERSION}"
```